test(form-builder): add workshop and unit tests for PTE

skogsmaskin opened the pull request:
### Description

This will add examples for the Portable Text Editor to the Workshop.

It will help us migrating to @sanity/ui and facilitate offline and more lightweight development on the editor.

<!--
What changes are introduced?
Why are these changes introduced?
What issue(s) does this solve? (with link, if possible)
-->

Changes to the code:

* CSS files for the editor had to be renamed to `*.module.css` to get Vite happy.
* Some types had to be exported as type also.
* Removed unused props in various PTE components.

Changes to tests:
* Restored disabled unit test for the PTE using test library.
* Removed a unnecessary disabled test in the PTE.

### What to review

That the types export is ok most of all.

<!--
What steps should the reviewer take in order to review?
What parts/flows of the application/packages/tooling is affected?
Add a test script, if that makes sense.
-->

### Notes for release

Don't think it needs to be mentioned as it's a dev env thing.

<!--
A description of the change(s) that should be used in the release notes.
-->

mariuslundgard commented on packages/@sanity/portable-text-editor/src/editor/PortableTextEditor.tsx:
```diff
@@ -27,7 +27,7 @@ export const defaultKeyGenerator = () => randomKey(12)
 const debug = debugWithName('component:PortableTextEditor')
 
-type Props = {
+export type Props = {
```
Rename  to `PortableTextEditorProps` since this is now exported?

mariuslundgard reviewed (approved):
LGTM – just one question in the comments 👍

skogsmaskin commented:
Cleaned up the commit history. Squashed everything `test(form-builder): workshop hmhm` together to `
test(form-builder): add workshop for the PTE`

Also added 64d98ec to satisfy the linter (unused prop).
